xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1406,3 +1407,19 @@ void CAdvancedSettings::setExtraLogLevel(const std::vector<CVariant> &components
     m_extraLogLevels |= static_cast<int>(it->asInteger());
   }
 }
+
+std::string CAdvancedSettings::GetMusicExtensions() const
```

Comment:
Should the result of this method be cached? From a quick glance it's called from multiple spots in the codebase and caching it might make sense? Cache could be invalidated/regenerated when a related addon has be installed/uninstalled or it's configuration has been changed.